api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.7.x.xml

```diff
@@ -100,4 +100,131 @@
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="concept_id" type="INT">
+				<constraints nullable="false"/>
+			</column>
+			<column name="criteria" type="VARCHAR(255)">
+				<constraints nullable="true"/>
+			</column>
+			<column name="hi_absolute" type="DOUBLE"/>
+			<column name="hi_critical" type="DOUBLE"/>
+			<column name="hi_normal" type="DOUBLE"/>
+			<column name="low_absolute" type="DOUBLE"/>
+			<column name="low_critical" type="DOUBLE"/>
+			<column name="low_normal" type="DOUBLE"/>
+			<column name="date_changed" type="TIMESTAMP"/>
+			<column name="date_created" type="TIMESTAMP"/>
+			<column name="creator" type="INT"/>
+			<column name="retired" type="BOOLEAN" defaultValue="false"/>
+			<column name="retired_by" type="INT"/>
+			<column name="retire_reason" type="VARCHAR(255)"/>
+			<column name="date_retired" type="TIMESTAMP"/>
+			<column name="changed_by" type="INT"/>
+			<column name="uuid" type="char(38)">
+				<constraints nullable="false" unique="true" />
+			</column>
+		</createTable>
+
+		<!-- foreign key constraints -->
+		<addForeignKeyConstraint
+			baseTableName="concept_reference_range"
+			baseColumnNames="concept_id"
+			constraintName="fk_concept_reference_range_concept"
+			referencedTableName="concept"
+			referencedColumnNames="concept_id"/>
+
+		<addForeignKeyConstraint
+			baseTableName="concept_reference_range"
+			baseColumnNames="creator"
+			constraintName="concept_reference_range_creator_fk"
+			referencedTableName="users"
+			referencedColumnNames="user_id"/>
+
+		<addForeignKeyConstraint
+			baseTableName="concept_reference_range"
+			baseColumnNames="changed_by"
+			constraintName="concept_reference_range_changed_by_fk"
+			referencedTableName="users"
+			referencedColumnNames="user_id"/>
+
+		<addForeignKeyConstraint
+			baseTableName="concept_reference_range"
+			baseColumnNames="retired_by"
+			constraintName="concept_reference_range_retired_by_fk"
+			referencedTableName="users"
+			referencedColumnNames="user_id"/>
+	</changeSet>
+
+	<changeSet author="Isaiah (generated)" id="202407231820-1">
+		<preConditions onError="HALT" onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="obs_reference_range"/>
+			</not>
+		</preConditions>
+		<createTable tableName="obs_reference_range" >
+			<column name="obs_reference_range_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="obs_id" type="INT">
+				<constraints nullable="false"/>
+			</column>
+			<column name="hi_absolute" type="DOUBLE"/>
+			<column name="hi_critical" type="DOUBLE"/>
+			<column name="hi_normal" type="DOUBLE"/>
+			<column name="low_absolute" type="DOUBLE"/>
+			<column name="low_critical" type="DOUBLE"/>
+			<column name="low_normal" type="DOUBLE"/>
+			<column name="date_changed" type="TIMESTAMP"/>
+			<column name="date_created" type="TIMESTAMP"/>
+			<column name="creator" type="INT"/>
```

Comment:
Obs are not metadata and hence not retire-able.  See Data vs Metadata https://archive.flossmanuals.net/openmrs-guide/openmrs-information-model.html